libs/types/src/logs_api.ts

```diff
@@ -0,0 +1,10 @@
+import { Result } from '@votingworks/basics';
```

Comment:
Instead of defining these types in libs/types, the pattern recommended for grout is to colocate them with the code that creates the API.

Also, you shouldn't define an interface for the API, but rather let grout infer it from the API object itself.

See https://github.com/votingworks/vxsuite/pull/4215